packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -16,11 +16,13 @@
 
 from ni_measurement_plugin_sdk_generator.client._support import (
     camel_to_snake_case,
+    can_generate_name,
     get_configuration_metadata_by_index,
     get_configuration_parameters_with_type_and_default_values,
     get_measurement_service_stub,
     get_output_metadata_by_index,
     get_output_parameters_with_type,
+    get_service_class_list,
```

Comment:
@mshafer-NI is there a point where it's cleaner just to import the whole module instead of all of the functions?

Reply:
We can use `from ni_measurement_plugin_sdk_generator.client._support import *`, which allows importing all the modules. However, it is generally not recommended. See [this](https://stackoverflow.com/questions/2360724/what-exactly-does-import-import)

Reply:
I hope we can procced with explicit import. @dixonjoel